analyzer/src/test/java/com/novoda/sqlite/StringUtilTest.java

```diff
@@ -0,0 +1,21 @@
+package com.novoda.sqlite;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class StringUtilTest {
+    @Test
+    public void shouldCamelizeColumnNames(){
+        assertEquals("ColumnId", StringUtil.camelify("COLUMN_ID"));
+        assertEquals("Columnid", StringUtil.camelify("columnID"));
+        assertEquals("Columnid", StringUtil.camelify("COLUMNID"));
```

Comment:
Wouldn't you want it to equal `ColumnId`?

also isn't camel case lower first letter? `columnId` 

It would be better if all three of these where separate tests for faster feedback, same with the below